src/test/java/org/broadinstitute/hellbender/tools/walkers/vqsr/GatherTranchesIntegrationTest.java

```diff
@@ -19,7 +19,7 @@
     @Test
     public void testCombine2Shards() throws Exception {
         final File recal1 = new File(testDir + "snpTranches.scattered.txt");  //this is the output of VariantRecalibratorIntegrationTest.testVariantRecalibratorSNPscattered
-        final File recal2 = new File(testDir + "snpTranches.scattered.2.txt"); //this is the output of running the equivalent commandline as above outside of the integration test :-/
```

Comment:
Not really sure why the original snpTranches.scattered.txt and snpTranches.scattered.2.txt test files were different. I would take this to indicate that running VariantRecalibratorIntegrationTest.testVariantRecalibratorSNPscattered in/outside of IntelliJ gave different results?

Just went ahead and changed this to be two copies of the VariantRecalibratorIntegrationTest.testVariantRecalibratorSNPscattered output. Although we could decouple the tests as well...

Reply:
Yep, that's what happened.  It's possible I was just trying to document the existence of some non-determinism.

Reply:
Gotcha. Hmm, I would be surprised if running *only* the testVariantRecalibratorSNPscattered test through IntelliJ produced different results from running at the command line. However, I wouldn't be surprised if other tests were run in conjunction with testVariantRecalibratorSNPscattered, since that will change the state of the Utils.getRandomGenerator() (especially since the test order will be random as well).

In general, I think there's no benefit to using a global random generator. The production code in the new tools is more careful about this sort of thing, and I'll make the test code equally so.